src/sections/CTASection/CTASection.tsx

```diff
@@ -10,6 +10,7 @@ import media from '../../globalStyles/media';
 import ctaBackground from '../../images/cta-background.webp';
 
 const HighlightPanel = styled.div`
+  padding: 0px 0px 100px 0px !important;
   h3 {
     color: ${({ theme }) => theme.color.black};
     margin: 0;
```

Comment:
1. Is `!important` needed?
2. Doc says no black color should be present. I think we should fix it everywhere where it's still used
3. I think this `h3` child is legacy and no longer needed - needs verification

Reply:
1. No, I copied it from another component and forgot to remove 🙈
2. Like you saw later, in reality this is not black color. But you are right that this is super confusing, I will change its name. 
3. Yes, it's no longer needed, I deleted it.